.env.example

```
@@ -10,3 +10,4 @@ PUBLIC_PRICES_API_URL=https://prices.openfoodfacts.net
 PUBLIC_ROBOTOFF_URL=https://robotoff.openfoodfacts.net
 PUBLIC_IMAGES_URL=https://images.openfoodfacts.net
 PUBLIC_NUTRIPATROL_URL=https://nutripatrol.openfoodfacts.net
+PUBLIC_SENTRY_DSN=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)